### PR TITLE
Update raddb/mods-available/{eap,eap_inner}

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: raddb/mods-available/eap [hunk 1]
```diff
@@ -1,58 +1,71 @@
...
 
 #######################################################################
 #
+#  = EAP Module
+#
+#  The `eap` module takes care of all EAP authentication as described in RFC 3579.
+#
+#  [WARNING]
+#  ====
 #  Whatever you do, do NOT set 'Auth-Type := EAP'.  The server is smart enough
```
Comment (arr2036) [inline raddb/mods-available/eap]: Maybe add a note saying where it could potentially be set or not set.

File: raddb/mods-available/eap [hunk 2]
```diff
@@ -67,111 +80,135 @@ eap {
...
 	#    make
 	#
-	#  These test certificates SHOULD NOT be used in a normal deployment.
+	#  These test certificates *SHOULD NOT* be used in a normal deployment.
 	#  They are created only to make it easier to install the server, and to
-	#  perform some simple tests with EAP-TLS, TTLS, or PEAP.
+	#  perform some simple tests with `EAP-TLS`, `TTLS`, or `PEAP`.
 	#
-	#  Note that you should NOT use a globally known CA here! e.g. using a
+	#  NOTE: You should NOT use a globally known CA here! e.g. using a
```
Comment (arr2036) [inline raddb/mods-available/eap]: The contents of this note should be removed or moved.  Cert verification no longer works like this.

File: raddb/mods-available/eap [hunk 3]
```diff
@@ -318,28 +380,34 @@ eap {
 		#
 		ca_file = ${cadir}/rsa/ca.pem
```
Comment (arr2036) [inline raddb/mods-available/eap]: This is where the note about global CAs should be.  Otherwise it's confusing.

**Comment by jpereira** [comment]
@arr2036 I did those changes as `fixup`. if you are ok, let me know to do a manual `autosquash` or we can  do that during the github merge/squash.

**Comment by arr2036** [comment]
Maybe do the autosquash at the end.  We can wait for the other fixes.  You saw the other three right?

File: raddb/mods-available/eap_inner [hunk 4]
```diff
@@ -1,87 +1,78 @@
...
 	#
 	type = md5
 	type = gtc
 	type = mschapv2
 	type = tls
 
 	#
-	#  MD5 is OK to use the inner tunnel, as keying material is provided by
-	#  the outer EAP method.
+	#  md5 { ... }:: See the `eap` module for common configuration explanation.
```
[the remark this answers is not in the document]
Comment (jpereira) [reply raddb/mods-available/eap_inner]: image
like this @arr2036 ?

**Comment by jpereira** [comment]
Hi @arr2036, thank you for the points. so, I updated as the below sample.

* raddb/mods-available/eap
image

* raddb/mods-available/eap_inner
image

**Comment by arr2036** [comment]
Looks good, thanks!

**Pull request merged**